tests/test_odoo_updates.py

```diff
@@ -125,6 +125,59 @@ def test_12_branches_to_screen(self):
         odoo_updates.branches_to_screen(branches)
         # TODO how to test this function?
 
+    def test_13_get_fields(self):
+        res = odoo_updates.get_fields('test_original')
+        self.assertEquals(len(res), 4)
+        self.assertIsInstance(res, list)
+        for record in res:
+            self.assertIsInstance(record, dict)
+
+    def test_14_compare_fields(self):
+        original = odoo_updates.get_fields('test_original')
+        updated = odoo_updates.get_fields('test_updated')
+        res = odoo_updates.compare_fields(original, updated)
+        validate(res, self.schema)
+        self.assertEquals(len(res['added']), 1)
+        self.assertEquals(len(res['deleted']), 1)
+        self.assertEquals(len(res['updated']), 3)
+        self.assertNotEqual(res['updated'][0]['original'],
+                            res['updated'][0]['modified'])
+        self.assertNotEqual(res['updated'][1]['original'],
+                            res['updated'][1]['modified'])
+        self.assertNotEqual(res['updated'][2]['original'],
+                            res['updated'][2]['modified'])
+        self.assertEquals(res['updated'][0]['model'], 'test_module1')
+        self.assertEquals(res['updated'][1]['model'], 'test_module1')
+        self.assertEquals(res['updated'][2]['model'], 'test_module2')
```

Comment:
@mapuerta  Make the tests using only 1 record per state (added, update, deleted) to make them shorter